docs/changelogs/v2.5.0.md

```diff
@@ -0,0 +1,117 @@
+- Only update last_used_at when connection count > 0 (#10808) (@sreya)
+- Update workspace cleanup flag names for template cmds (#10805) (@sreya)
+- Give SSH stdio sessions a chance to close before closing netstack (#10815) (@spikecurtis)
+- Preserve order of node reports in healthcheck (#10835) (@mtojek)
+- Enable FeatureHighAvailability if it is licensed (#10834) (@spikecurtis)
+- Skip autostart for suspended/dormant users (#10771) (@coadler)
+- Display explicit 'retry' button(s) when a workspace fails (#10720) (@Parkreiner)
+- Improve exit codes for agent/agentssh and cli/ssh (#10850) (@mafredri)
+- Detect and retry reverse port forward on used port (#10844) (@spikecurtis)
+- Document workspace filter query param correctly (#10894) (@Kira-Pilot)
+- Hide groups in account page if not enabled (#10898) (@Parkreiner)
+- Add spacing for yes/no prompts (#10907) (@f0ssel)
+- Numerical validation grammar (#10924) (@ericpaulsen)
+- Insert replica when removed by cleanup (#10917) (@f0ssel)
+- Update autostart context to include querying users (#10929) (@sreya)
+- Clear workspace name validation on field dirty (#10927) (@Kira-Pilot)
+- Redirect to new url after template name update (#10926) (@Kira-Pilot)
+- Do not allow selection of unsuccessful versions (#10941) (@f0ssel)
+- Parse username/workspace correctly on `coder state pull --build` (#10973) (#10974) (@spikecurtis)
+- Handle 404 on unknown top level routes (#10964) (@f0ssel)
+- FIX `UpdateWorkspaceDormantDeletingAt` interval out of range (#11000) (@coadler)
+- Create centralized PaginationContainer component (#10967) (@Parkreiner)
+- Use database for user creation to prevent flake (#10992) (@f0ssel)
+- Pass in time parameter to prevent flakes (#11023) (@f0ssel)
+- Respect header flags in wsproxy server (#10985) (@deansheather)
+- Update tailscale to include fix to prevent race (#11032) (@spikecurtis)
+- Disable prefetches for audits table (#11040) (@Parkreiner)
+- Increase default staleTime for paginated data (#11041) (@Parkreiner)
+- Display app templates correctly in build preview (#10994) (@Kira-Pilot)
+- Redirect unauthorized git users to login screen (#10995) (@Kira-Pilot)
+- Use unique workspace owners over unique users (#11044) (@f0ssel)
+- Stop updating agent stats from deleted workspaces (#11026) (@f0ssel)
+- Track JetBrains connections (#10968) (@code-asher)
+- Handle no memory limit in `coder stat mem` (#11107) (@f0ssel)
+- Provide helpful error when no login url specified (#11110) (@f0ssel)
+- Return 403 when rebuilding workspace with require_active_version (#11114) (@sreya)
+- Use provisionerd context when failing job on canceled acquire (#11118) (@spikecurtis)
+- Ensure we are talking to coder on first user check (#11130) (@f0ssel)
+- Prevent logging error for query cancellation in `watchWorkspaceAgentMetadata` (#10843) (@mafredri)
+- Keep workspace agent connection open after dial context (#10863) (@mafredri)
+- Fix a broken link (#10783) (@matifali)
```

Comment:
Do we normally leave these ambiguous?

Reply:
We usually delete non user facing commits. e.g.
> - Fix a broken link (#10783) (@matifali)

This could be removed.

Reply:
Fixed